src/library/basetracktablemodel.cpp

```diff
@@ -470,15 +470,13 @@ QVariant BaseTrackTableModel::composeCoverArtToolTipHtml(
     // Determine height of the cover art image depending on the screen size
     unsigned int absoluteHeightOfCoverartToolTip;
```

Comment:
Define and initialize after primaryScreen check

Reply:
Done in #3298.